pkg/gitprovider/aws-codecommit.go

```diff
@@ -47,6 +49,9 @@ func (g *AwsCodeCommitGitProvider) GetNamespaces() ([]*GitNamespace, error) {
 		SortBy: types.SortByEnumRepositoryName,
 	})
 	if err != nil {
+		if reqErr, ok := err.(awserr.RequestFailure); ok {
```

Comment:
Create a `FormatError` function as well here (and in the rest of the aws codecommit provider)